openedx/core/djangoapps/course_date_signals/handlers.py

```diff
@@ -1,11 +1,13 @@
 """Signal handlers for writing course dates into edx_when."""
 
 
+from datetime import timedelta
 import logging
 
 from django.dispatch import receiver
 from edx_when.api import FIELDS_TO_EXTRACT, set_dates_for_course
 
+from cms.djangoapps.contentstore.config.waffle_utils import custom_pls_is_active
```

Comment:
While you're in here, would you mind reordering the imports here? The xmodule.util.misc looks like it should be lower and I believe the xblock.fields should be below the edx_when line above. If you move the xblock.fields line, please also delete the lint amnesty comment since we don't want to ignore the wrong import order anymore